src/stories/collapsible.stories.js

```diff
@@ -0,0 +1,50 @@
+import React from "react"
+import StoryContainer from "./storyContainer"
+import Collapsible from "../components/collapsible"
+import { action } from "@storybook/addon-actions"
+
+export default {
+  title: "Collapsible",
+  component: Collapsible,
+  args: {
+    isInitiallyCollapsed: true,
+    renderToggle: (options) => options,
+    onCollapseStateChange: (isCollapsed) => isCollapsed,
+    collapseIconSize: 32,
+    opacityOnHover: true,
```

Comment:
this is part of our default link style. I would quickly align with the designer. we might want either or, but not switch it on/off per case. this is a great moment in time to align the hover style of all collapsibles instead of allowing both variants

Reply:
this is also only because of the dealerhub overview page. I discussed this one with @lkappeler . The `a` tag has a default hover styling with `opacity: 60`. it looks a bit strange if the whole component gets transparent on the listing overview

Reply:
I'm sure there's a way to disable the default opacity on the parent DOM element instead of leaking this project-specific weirdness into the components-pkg. Can we try to get rid of this extra prop?

Reply:
unfortunately, CSS does not support parent selectors, only sibling and child. I can try to use a `ref` and then set the style dependent if the ref on the child is set or not. But this probably makes the code more complex than it is now

Reply:
I meant... if there's a default opacity, we might be able to override it on the parent. the most naive approach in pseudocode:

```
opacity-0 !important (add a new wrapper that overrides the behavior of the child)
  opacity-60 (I assume this is what's in the children)
```

as this thread starts to get long, it might be a good time to quickly align in a call. just ping me!

Reply:
We just discussed this on the phone. Opacity is relative to the parent, so we can't override it on the child. We get the feeling that the current approach is the most descriptive we can come up with. I'm fine moving on, despite that the prop feels a bit out-of-place and leaks project-specific global css into the component. Let's roll with this if we see no other way out